template/web/src/hooks/MockABI.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * This ABI is trimmed down to just the functions we expect to call for the
+ * sake of minimizing bytes downloaded.
+ */
+const abi = [
```

Comment:
let's move all tests into the `test` folder a `src/hooks/test` so also the Mock goes there.